addons/mrp/security/ir.model.access.csv

```diff
@@ -63,3 +63,5 @@ access_mrp_consumption_warning,access.mrp.consumption.warning,model_mrp_consumpt
 access_mrp_consumption_warning_line,access.mrp.consumption.warning.line,model_mrp_consumption_warning_line,mrp.group_mrp_user,1,1,1,0
 access_mrp_immediate_production,access.mrp.immediate.production,model_mrp_immediate_production,mrp.group_mrp_user,1,1,1,0
 access_mrp_immediate_production_line,access.mrp.immediate.production.line,model_mrp_immediate_production_line,mrp.group_mrp_user,1,1,1,0
+access_mrp_workcenter_tag_manager,access.mrp.workcenter.tag,model_mrp_workcenter_tag,mrp.group_mrp_manager,1,1,1,1
```

Comment:
Inverse line with the next one, we prefer always to sort by (model,) access (more access always in bottom - after sorting by model). (Yes it is not respected above, but I have a very old task to clean ACL and I don't want to have more diff :pray: )